canonical/microstack.run
# [Sunbeam GA Launch] /index revamp

## Done

- Updated copy as per [doc](https://docs.google.com/document/d/1-QO5zAclj469qGvFmNJBjQ_qqI_T_uliwj6mAo8ZbCs/edit#)
- NOTE: Will include [this CTA](https://docs.google.com/document/d/1-QO5zAclj469qGvFmNJBjQ_qqI_T_uliwj6mAo8ZbCs/edit?disco=AAAAu0CfWYo) when the pdf becomes available 
## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8039/
- See that the requested changes have been made

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-3619
